src/test/scala/com/thoughtworks/EnableWithArtifactTest.scala

```diff
@@ -0,0 +1,55 @@
+package com.thoughtworks
+
+import org.scalatest._
+import enableIf._
+
+import scala.util.control.TailCalls._
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+
+
+/**
+ * @author 沈达 (Darcy Shen) &lt;sadhen@zoho.com&gt;
+ */
+class EnableWithArtifactTest extends AnyFreeSpec with Matchers {
+  "Test if we are using quasiquotes explicitly" in {
+
+    object ExplicitQ {
+
+      @enableIf(classpathMatchesScalaArtifact("quasiquotes", "2.1.1"))
```

Comment:
```suggestion
      @enableIf(classpathMatchesArtifact(crossScalaBinaryVersion("quasiquotes"), "2.1.1"))
```

"Scala artifact" is not a good name because there are more than one way to add the suffix for a scala artifact. I would add some helper functions like `crossScalaBinaryVersion` and `crossScalaFullVersion`, corresponding to [the `CrossVersion` API in sbt](https://www.scala-sbt.org/1.x/docs/Cross-Build.html), for more accurate terminology.

Reply:
Thanks, improved.